packages/vite/src/node/plugins/worker.ts

```diff
@@ -287,7 +287,9 @@ export function webWorkerPlugin(config: ResolvedConfig): Plugin {
             export default function WorkerWrapper() {
               const objURL = blob && (window.URL || window.webkitURL).createObjectURL(blob);
```

Comment:
if setting `base64` is not need to createObjectURL

Reply:
updated. Separate the blob and base64 code to make it more readable for human.